.github/workflows/ci.yaml

```diff
@@ -131,6 +131,9 @@ jobs:
     if: ${{ github.ref == 'refs/heads/master' && github.event_name == 'push'}}
     needs: build-dse
     runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        jdk-version: [jdk8, jdk11]
```

Comment:
Do we need the jdk8 builds?

Reply:
We don't necessarily, but I believe the cass-operator default DSE image is a JDK8 based image. The official DSE images have a `-java11` suffix to the image tag for JDK11 based images, while the JDK8 based images do not have a suffix for the tag.

Reply:
Also, the DSE images aren't official, just used for testing at the moment. But it doesn't hurt to build both as the official build process will.